docs/antrea-agent-simulator.md

```diff
@@ -0,0 +1,74 @@
+[toc]
```

Comment:
Remove this? It appears in the rendered page

Reply:
btw if you don't want to include a TOC for a doc, you can just add it to `hack/.notableofcontents`